api/envoy/config/route/v3/route_components.proto

```diff
@@ -457,6 +457,28 @@ message RouteMatch {
     // Note that CONNECT support is currently considered alpha in Envoy.
     // [#comment:TODO(htuch): Replace the above comment with an alpha tag.
     ConnectMatcher connect_matcher = 12;
+
+    // Supports Google url_template matching for ":path" header.
+    // This match is always case sensitive.
+    // Its syntax is defined at `http.proto <https://github.com/googleapis/googleapis/blob/f829b1334cce86aa3738f3c0698d814b56664445/google/api/http.proto#L208>`_.
+    //
+    // Syntax highlights:
+    //
+    // .. code-block:: text
+    //
+    // * wildcard `*`: matches only 1 path segment.
+    // * double wildcard `**`: matches 0 or more path segments.
```

Comment:
Can you explain about using this only at end of path?

Reply:
you mean double wildcard **.   
not necessary at the end of path.  it can be in the middle e.g.   /foo/**/tail.

Reply:
From `http.proto` there is:
```
The syntax `**` matches
// zero or more URL path segments, which must be the last part of the URL path
// except the `Verb`.
```
Is this a place that the gRPC-JSON matcher is inconsistent with that documentation?

Reply:
Yes, it is inconsistent between implementation and that documentation. Checking with Hong to see if we should correct the documentation.

Reply:
Hong agreed to update the doc to be consistent with implementation.